core/src/components/input/test/basic/input.e2e.ts

```diff
@@ -185,6 +223,28 @@ configs({ modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
       const container = page.locator('#container');
       await expect(container).toHaveScreenshot(screenshot(`input-clear-button-focused`));
     });
+
+    test('should not have visual regressions when user dbclicks', async ({ page, browserName }) => {
+      if (browserName === 'firefox' || browserName === 'webkit') {
+        //TODO: Figure out why it's not working for webkit & firefox
+        test.skip();
+      }
```

Comment:
Once I used the native input to click on then the Firefox and Safari tests were passing as expected.

```suggestion
```